### PR TITLE
Udpate docs

### PR DESCRIPTION
**Pull request by IgorSusmelj**
### Changes

- Add results for imagenette benchmark to the python script
- Increase the resolution of the benchmark output
- Add softmax to the pizza tutorial to make it work with newer PyTorch lightning versions

### CODE REVIEW

**Review by MalteEbner — APPROVED** [review]
This looks good to me.
